src/core/lib/iomgr/tcp_server_posix.cc

```diff
@@ -388,7 +397,7 @@ static grpc_error_handle clone_port(grpc_tcp_listener* listener,
     sp->fd = fd;
     sp->emfd = grpc_fd_create(
         fd,
-        absl::StrFormat("tcp-server-listener:%s/clone-%d", addr_str.c_str(), i)
+        absl::StrFormat("tcp-server-listener:%s/clone-%d", addr_str->c_str(), i)
```

Comment:
No need to call `addr_str->c_str()` here, since `absl::StrFormat()` can deal with `std::string` arguments.  Just use `*addr_str` instead.

Reply:
Done.